chaos_genius/core/utils/data_loader.py

```diff
@@ -260,6 +260,17 @@ def _preprocess_df(self, df):
             .dt.tz_localize(None)
         )
 
+        # check if preaggregated count column and metric column are the same
+        # if yes, then duplicate the count column and store it as an object variable
+        self._preaggregated = self.connection_info["connection_type"] == "Druid"
+        self._preaggregated_count_col = self.kpi_info["count_column"]
+        if (
+            self._preaggregated
+            and self._preaggregated_count_col == self.kpi_info["metric"]
+        ):
+            self.pre_aggregated_count_column = randomword(10)
```

Comment:
It would be good to have a prefix for this that indicates that it's the duplicate count column, just in case there are any exceptions that print it.

```suggestion
            self.pre_aggregated_count_column = f"count_col_duplicate_{randomword(10)}"
```